modules/InboundEmail/InboundEmail.php

```diff
@@ -409,31 +410,33 @@ public function checkWithPagination($offset = 0, $pageSize = 20, $order = array(
 
         $emailHeaders = json_decode(json_encode($emailHeaders), true);
         // get attachment status
-        foreach ($emailHeaders as $i=> $emailHeader) {
+        foreach ($emailHeaders as $i => $emailHeader) {
 
-            $structure = imap_fetchstructure($this->conn,  $emailHeader['msgno'], FT_UID);
+            $structure = imap_fetchstructure($this->conn, $emailHeader['msgno'], FT_UID);
 
-            if(isset($structure->parts[0]->parts))
-            {
+            if (isset($structure->parts[0]->parts)) {
                 // has attachment
                 $emailHeaders[$i]['has_attachment'] = true;
-            } else{
+            } else {
                 // no attachment
                 $emailHeaders[$i]['has_attachment'] = false;
             }
         }
 
 
-        usort($emailHeaders, function($a, $b) use($sortCRM){  // defaults to DESC order
-            if($a[$sortCRM] === $b[$sortCRM]) {
+        usort($emailHeaders, function ($a, $b) use ($sortCRM) {  // defaults to DESC order
+            if ($a[$sortCRM] === $b[$sortCRM]) {
                 return 0;
-            } else if($a[$sortCRM] < $b[$sortCRM]) {
+            } elseif ($a[$sortCRM] !== $b[$sortCRM] && $a[$sortCRM] < $b[$sortCRM]) {
                 return 1;
-            } else {
+            } elseif ($a[$sortCRM] !== $b[$sortCRM]) {
```

Comment:
Whats the scenario if it doesn't hit any of these? wouldn't it be easier just stating 'else' than another elseif